testhelpers/docker_registry.go

```diff
@@ -135,10 +140,14 @@ func (r *DockerRegistry) Start(t *testing.T) {
 	r.authnHandler = ReadOnly(r.regHandler)
 	if r.username != "" {
 		if r.customPrivileges != nil {
+			// wrap registry handler with basic auth
+			r.authnHandler = BasicAuth(r.regHandler, r.username, r.password, "registry")
 			// wrap registry handler with custom handler
-			r.regHandler = Custom(r.regHandler, r.customPrivileges)
+			r.authnHandler = Custom(r.authnHandler, r.regHandler, r.customPrivileges)
```

Comment:
```suggestion
			r.authnHandler = Custom(basicAuthHandler, r.regHandler, r.customPrivileges)
```

Reply:
(I think this makes it a little more clear)

Reply:
I agreed, applied in the following [commit](https://github.com/buildpacks/imgutil/pull/126/commits/f19016e4dd378dcd0531ef1d351626730dd1ccb7)